### ISSUE TITLE
Was this project dropped?

### ISSUE THREAD

**Issue by lleoha** [opensatelliteproject/metlrpt#2]
I see that it is marked as WIP but the last commit was c.a. one year ago.
Was the work on this LRPT decoder dropped? Is project dead?

**Comment by racerxdl** [comment]
Nope, I just stopped for now to finish the changes on other OSP features since it was taking some time to make the JPG decoder for Meteor. But I still plan to finish that.

**Comment by kuttkutt** [comment]
Hello racerxdl, 

that would be awesome!

**Comment by lleoha** [comment]
@racerxdl Lucas, do you accept pull request for this project or is it going to be one-man project only? :) I think I can help you with this kind of stuff if you want.

**Comment by racerxdl** [comment]
@lleoha sure, everyone can contribute to OSP :D - Feel free to make a pull request ^^

**Comment by lleoha** [comment]
@racerxdl one more thing...
Do you have any documentation/specification on how images are compressed/encoded before putting them into CCSDS packets? This is the only missing piece I cannot find :(.
I know how those packets are packed in other layers, then RS coded, convolution coded and so on, but the very essential thing of how images are compressed is unknown for me. I would appreciate you sharing me the spec if you have one. Thanks!
Interesting thing is that I was able to find everything for LRIT/HRIT but I am not able to find for LRPT.

**Comment by racerxdl** [comment]
Yes, just a second I have some specs here. Let me find.

**Comment by racerxdl** [comment]
http://meteor.robonuka.ru/for-experts/soft/ - I was trying to use the info there, which is probably the best that can be found for Meteor.

**Comment by lleoha** [comment]
thanks!

-- 
Best Regards,
Mateusz Kramarczyk
